File: SFAN/frontend/bump.py
import os
import re

html_files = ['dashboard_broker.html', 'dashboard_individual.html', 'dashboard_policyholder.html']
for f in html_files:
    if os.path.exists(f):
        with open(f, 'r', encoding='utf-8') as file:
            content = file.read()
        content = re.sub(r'(\.js\?v=)\d+', r'\g<1>99', content)
        with open(f, 'w', encoding='utf-8') as file:
            file.write(content)

js_files = ['dashboard/main_broker.js', 'dashboard/main_individual.js', 'dashboard/main_policyholder.js']
for f in js_files:
    if os.path.exists(f):
        with open(f, 'r', encoding='utf-8') as file:
            content = file.read()
        content = re.sub(r'(\.js\?v=)\d+', r'\g<1>99', content)
        with open(f, 'w', encoding='utf-8') as file:
            file.write(content)

File: SFAN/frontend/bump.js
const fs = require('fs');

const files = [
    'dashboard_broker.html',
    'dashboard_individual.html',
    'dashboard_policyholder.html',
    'dashboard/main_broker.js',
    'dashboard/main_individual.js',
    'dashboard/main_policyholder.js'
];

files.forEach(f => {
    if (fs.existsSync(f)) {
        let content = fs.readFileSync(f, 'utf-8');
        content = content.replace(/\.js\?v=\d+/g, '.js?v=99');
        fs.writeFileSync(f, content, 'utf-8');
    }
});
